### PR TITLE
fix(clerk-js): Do not distort application logo

### PR DESCRIPTION
**Pull request by LekoArts**
## Description

- Removes an unnecessary `object-fit` from `logoBox`
- Add `object-fit: contain` to `logoImage` so that image is not distorted. Aligns with what https://developer.mozilla.org/en-US/docs/Learn/CSS/Howto/Fill_a_box_with_an_image explains

Fixes SDK-1781

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by LekoArts** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @LekoArts, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-1oeey1i5m.clerkstage.dev) | Jun 03, 2024 12:12 PM |

File: packages/clerk-js/src/ui/elements/ApplicationLogo.tsx [hunk 1]
```diff
@@ -49,7 +49,8 @@ export const ApplicationLogo = (props: ApplicationLogoProps) => {
       sx={{
         display: loaded ? 'inline-block' : 'none',
         height: '100%',
-        width: 'auto',
+        width: '100%',
+        objectFit: 'contain',
```
Comment (alexcarpenter) [inline packages/clerk-js/src/ui/elements/ApplicationLogo.tsx:53]: This means that if they upload a photo that is a rectangle, there will be awkward negative space where the logo is rendered. Is that expected/desired? Also seeing cover usage in https://github.com/clerk/javascript/commit/63373bf218d7e20932728f8908f90316d34dec07

Comment (LekoArts) [reply packages/clerk-js/src/ui/elements/ApplicationLogo.tsx:53]: Do you have a visual example of what you mean? I think `cover` makes sense for an avatar

**Review by alexcarpenter — APPROVED** [review]

**Comment by desiprisg** [comment]
Isn't this a breaking change?

**Comment by LekoArts** [comment]
Depends on how you customize your styles. With default styles, nothing changes. With minor edits, nothing changes. If you overwrote things like width/height, things might change

**Pull request merged**